idl/gossip_digest.idl.hh

```diff
@@ -56,6 +56,7 @@ class gossip_digest_syn {
     sstring get_cluster_id();
     sstring get_partioner();
     utils::chunked_vector<gms::gossip_digest> get_gossip_digests();
+    std::optional<sstring> get_group0_id()[[version 5.4]];
```

Comment:
Because `cluster_id` uses `sstring`. I now see it is not an ID but a cluster name (gossiper stores it as `cluster_name`). I will change that to `UUID`.